app/services/mailer.js

```diff
@@ -24,16 +24,32 @@ const getTransporter = () => {
   return transporter;
 };
 
+const sendMailToUser = (user, { subject, text }) =>
+  getTransporter().sendMail({
+    from: mailer.from,
+    to: user.email,
+    subject,
+    text
+  });
+
 exports.sendWelcomeEmail = user => {
   info(`mailer.sendWelcomeEmail to user ${user.id}`);
-  return getTransporter()
-    .sendMail({
-      from: mailer.from,
-      to: user.email,
-      subject: 'Bienvendio a Weeter!',
-      text: `Hola ${user.firstName}. Tu cuenta ha sido creada correctamente en Weeter`
-    })
-    .catch(({ message }) => {
-      throw externalServiceError(`Error when sending welcome email to user ${user.id}. Error: ${message}`);
-    });
+  return sendMailToUser(user, {
+    subject: 'Bienvendio a Weeter!',
+    text: `Hola ${user.firstName}. Tu cuenta ha sido creada correctamente en Weeter`
```

Comment:
Messages are usually set in env vars. That way, if we see a typo ('bienvendio'), or we want to change the message, we can change the env and re-deploy, without having to go through the entire pr proccess.